* src/qemu/qemu_conf.c: Report VIR_ERR_CONFIG_UNSUPPORTED for
  unsupported video adapters
---
 src/qemu/qemu_conf.c |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/qemu/qemu_conf.c b/src/qemu/qemu_conf.c
index 84b7067..b922907 100644
--- a/src/qemu/qemu_conf.c
+++ b/src/qemu/qemu_conf.c
@@ -4423,7 +4423,7 @@ int qemudBuildCommandLine(virConnectPtr conn,
             } else {
                 const char *vgastr = qemuVideoTypeToString(def->videos[0]->type);
                 if (!vgastr || STREQ(vgastr, "")) {
-                    qemuReportError(VIR_ERR_INTERNAL_ERROR,
+                    qemuReportError(VIR_ERR_CONFIG_UNSUPPORTED,
                                     _("video type %s is not supported with QEMU"),
                                     virDomainVideoTypeToString(def->videos[0]->type));
                     goto error;
@@ -4449,8 +4449,8 @@ int qemudBuildCommandLine(virConnectPtr conn,
                 break;
 
             default:
-                qemuReportError(VIR_ERR_INTERNAL_ERROR,
-                                _("video type %s is not supported with QEMU"),
+                qemuReportError(VIR_ERR_CONFIG_UNSUPPORTED,
+                                _("video type %s is not supported with this QEMU"),
                                 virDomainVideoTypeToString(def->videos[0]->type));
                 goto error;
             }
-- 
1.6.6.1
